packages/teams-js/src/public/constants.ts

```diff
@@ -49,15 +61,23 @@ export enum HostName {
   teamsModern = 'TeamsModern',
 }
 
-// Ensure these declarations stay in sync with the framework.
+/** FrameContexts provides information about the context in which the app is running within the host. */
```

Comment:
>FrameContexts provides information about the context in which the app is running within the host. [](http://example.com/codeflow?start=4&length=97)

Can you say more about how a developer might be expected to use this frame context information?